addons/storyshots/storyshots-core/README.md

```diff
@@ -185,6 +185,121 @@ initStoryshots({
 })
 ```
 
+
+### StoryShots for async rendered components
+
+You can make use of [Jest done callback](https://jestjs.io/docs/en/asynchronous) to test components that render asynchronously. This callback is passed as param to test method passed to `initStoryshots(...)` when the `asyncJest` option is given as true.
```

Comment:
I've just updated this PR with changes you requested @igor-dv .

By the way, I've used just `async` first but then I've opted for `asyncJest` to avoid issues with syntax _highlightings_ that were assuming it was the `async` ES keyword